client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -30,8 +30,10 @@
     private static final String APPIUM_ANDROID_PLATFORM_TYPE = "ANDROID";
     private static final String APPIUM_IOS_PLATFORM_TYPE = "iOS";
     private static final String MOBILE_NODE_TYPE = "mobileNodeType";
-    private static final String APPIUM_LOCALE_TYPE = "locale";
-    private static final String APPIUM_LANGUAGE_TYPE = "language";
+    private static final String MOBILE_APP_MAIN_ACTIVITY = "appActivity";
+    private static final String MOBILE_APP_WAIT_ACTIVITY = "appWaitActivity";
+    private static final String MOBILE_APP_PACKAGE = "appPackage";
+    private static final String MOBILE_APP_WAIT_PACKAGE = "appWaitPackage";
```

Comment:
Can you please point to some documentation on these 4 capabilities? I'm trying to understand their significance. Thanks!